src/workloads/execution/CreateBigIndex.yml

```diff
@@ -23,21 +23,30 @@ Actors:
   - &Nop {Nop: true}
   - *Nop
   - *Nop
+  - *Nop
+
+# Phase 2: Quiesce
+- Name: Quiesce
+  Type: QuiesceActor
+  Database: *db
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+  - *Nop
+  - *Nop
+  - *Nop
 
-# Phase 2: Lower the memory limit for spilling to disk so that it occurs more often.
+# Phase 3: Lower the memory limit for spilling to disk so that it occurs more often.
 - Name: Setup
   Type: RunCommand
   Threads: 1
   Phases:
   - *Nop
+  - *Nop
```

Comment:
In order to not change the results, I think you should probably keep the ```LimitIndexBuildMemoryUsageCommand``` in the second phase. 

Or at least run a patch to ensure the results are the same.

Reply:
As per our discussion, I've updated this so both`LimitIndexBuildMemoryUsageCommand ` and the QuiesceActor run in the same phase. 

I've also tested that none of the changes seem to impact performance: [PerfAnalyzer](https://performance-analyzer.server-tig.prod.corp.mongodb.com/perf-analyzer-viz/?evergreen_version=6389ee7aa4cf477e25f9f934&evergreen_base_version=sys_perf_552afb106b2cdc485db67087ce24e7d33e8cbabb&metrics_selection=Latency99thPercentile%7C%7Cops_per_sec%7C%7CLatency50thPercentile%7C%7COperationThroughput%7C%7CtpmC%7C%7CNewOrders%7C%7CDuration%7C%7Caverage_read_latency_us%7C%7C95th_read_latency_us%7C%7C99th_read_latency_us&percent_filter=0%7C%7C100&z_filter=0%7C%7C10)